core/src/main/java/org/infinispan/configuration/parsing/ConfigurationBuilderHolder.java

```diff
@@ -50,11 +50,7 @@ public ConfigurationBuilder getCurrentConfigurationBuilder() {
    }
 
    public ConfigurationBuilder getDefaultConfigurationBuilder() {
-      if (globalConfigurationBuilder.defaultCacheName().isPresent()) {
-         return namedConfigurationBuilders.get(globalConfigurationBuilder.defaultCacheName().get());
-      } else {
-         return null;
-      }
+       return globalConfigurationBuilder.defaultCacheName().map(namedConfigurationBuilders::get).orElse(null);
```

Comment:
Technically this is another lambda and out of scope of this PR imo.

Reply:
ah yes, haha I saw it and I thought in the refactoring but can remove it without any problem.
I forgot I did it here